content/en/docs/tasks/job/automated-tasks-with-cron-jobs.md

```diff
@@ -129,13 +127,13 @@ kubectl delete cronjob hello
 Deleting the cron job removes all the jobs and pods it created and stops it from creating additional jobs.
 You can read more about removing jobs in [garbage collection](/docs/concepts/workloads/controllers/garbage-collection/).
 
-## Writing a Cron Job Spec
+## Writing a CronJob Spec
```

Comment:
```suggestion
## Writing a Cron Job Spec
```

cc: @tengqm
As far as I know, "cron" is the right word and the best way to put this out would be in "cron job" which is how I have seen it being used elsewhere too.

The [dictionary meaning](https://www.google.com/search?q=meaning+of+cron) of cron seconds this and an example of using this in a sentence as shown in the default Google Dictionary is:

> "a cron job"

Also, here's a [Google Cloud](https://cloud.google.com/scheduler/docs/creating) documentation page using "cron job".

Reply:
I think there's a difference between saying Cron Job, CronJob, and cron Job or cron job

I think Cron Job should be avoided if possible (but totally a nit) because Cron shouldn't be a proper noun. A CronJob in Kubernetes is a way of running cron Jobs (Kubernetes Jobs on a repeatable schedule). 

So to me at least:

- Cron Job: no because Cron on its own shouldn't sound like a feature
- CronJob: use when referring to the Kubernetes CronJob object
- cron Job(s): use when talking about Kubernetes Jobs running on a repeatable schedule (that is, what you can use the CronJob object for)
- cron jobs: use when talking about generic jobs (Kubernetes or Unix or whatever) running on a repeatable schedule.

Idk if that's getting too pedantic though

Reply:
+1 to“cron job” for something running locally via cron and also for writing CronJob to represent the Kubernetes API it inspired.

In this page we should nearly always write CronJob.